### PR TITLE
[Fix] viz_surface

### PR DESCRIPTION
**Pull request by skoudoro**
The goal of this PR is to fix the Travis issue [here](https://travis-ci.org/fury-gl/fury/jobs/467619790#L3381)

I use this opportunity to move the `io` module from DIPY.

### CODE REVIEW

File: docs/examples/viz_surfaces.py [hunk 1]
```diff
@@ -15,16 +15,11 @@
...
-import fury.utils as ut_vtk
-from fury import window
-
-# Conditional import machinery for vtk
-# Allow import, but disable doctests if we don't have vtk
-from dipy.utils.optpkg import optional_package
-vtk, have_vtk, setup_module = optional_package('vtk')
+from fury.io import save_polydata, load_polydata
+from fury import window, utils
+from fury.utils import vtk
```
Comment (dmreagan) [inline docs/examples/viz_surfaces.py]: Sort imports

```python
from fury import utils, window
from fury.io import load_polydata, save_polydata
from fury.utils import vtk
```

File: fury/io.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        try:  # try to read as a normal obj
+            reader = vtk.vtkOBJReader()
+        except Exception:  # than try load a MNI obj format
+            reader = vtk.vtkMNIObjectReader()
+    else:
+        raise "polydata " + file_extension + " is not suported"
+
+    reader.SetFileName(file_name)
+    reader.Update()
+    # print(file_name + " Mesh " + file_extension + " Loaded")
```
Comment (dmreagan) [inline fury/io.py]: Remove commented line

File: fury/io.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    elif file_extension == "fib":
+        writer = vtk.vtkPolyDataWriter()
+    elif file_extension == "ply":
+        writer = vtk.vtkPLYWriter()
+    elif file_extension == "stl":
+        writer = vtk.vtkSTLWriter()
+    elif file_extension == "xml":
+        writer = vtk.vtkXMLPolyDataWriter()
+    elif file_extension == "obj":
+        raise Exception("mni obj or Wavefront obj ?")
```
Comment (dmreagan) [inline fury/io.py]: This exception asks the user to clarify which version of obj, but gives them no options for how to do so. All objs raise the exception.

**Comment by skoudoro** [comment]
This PR is ready to be merged, can you have a look @dmreagan @Garyfallidis

**Comment by Garyfallidis** [comment]
:)

**Pull request merged**